test/e2e/configuration_test.go

```diff
@@ -370,6 +372,12 @@ if (!"%s".equals(Jenkins.instance.systemMessage)) {
 	assert.NoError(t, err, logs)
 }
 
+func verifyServiceTargetPort(t *testing.T, jenkins *v1alpha2.Jenkins) {
+	jenkinsPod := getJenkinsMasterPod(t, jenkins)
```

Comment:
You have to fetch the actual two Kubernetes services created by the operator, no Jenkins pod.